airo-dataset-tools/test/test_pose.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+import numpy as np
+from airo_dataset_tools.pose import EulerAngles, Pose, Position
+
+
+def test_pose_simple():
+    pose = Pose(
+        position_in_meters=Position(x=1.0, y=2.0, z=3.0),
+        rotation_euler_XYZ_in_radians=EulerAngles(roll=np.pi / 4, pitch=-np.pi / 2, yaw=np.pi),
+    )
+
+    with open("pose.json", "w") as file:
```

Comment:
maybe better to use temporary file? https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html